Failing test: Security Solution Cypress.x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/rule_details/execution_log·cy·ts - Event log should display the execution log should display the execution log

A test failed on a tracked branch

```
AssertionError: Timed out retrying after 150000ms: Expected to find element: `[data-test-subj="filters-global-container"] [data-test-subj="querySubmitButton"]`, but never found it.
    at refreshPage (webpack:///./tasks/security_header.ts:31:5)
    at cy.waitUntil.interval (webpack:///./tasks/create_new_rule.ts:864:18)
    at evaluate (webpack:///./support/commands.js:74:19)
    at Context.waitUntil (webpack:///./support/commands.js:83:9)
```

First failure: [kibana-on-merge - main](https://buildkite.com/elastic/kibana-on-merge/builds/45777#018fbf7c-bff5-43a7-9113-5ec24d62aa03)

<!-- kibanaCiData = {"failed-test":{"test.class":"Security Solution Cypress.x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/rule_details/execution_log·cy·ts","test.name":"Event log should display the execution log should display the execution log","test.failCount":2}} -->

--- a/x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/rule_details/execution_log.cy.ts
+++ b/x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/rule_details/execution_log.cy.ts
@@ -12,14 +12,13 @@ import { login } from '../../../../tasks/login';
 import { visit } from '../../../../tasks/navigation';
 import { ruleDetailsUrl } from '../../../../urls/rule_details';
 import { createRule } from '../../../../tasks/api_calls/rules';
-import { waitForAlertsToPopulate } from '../../../../tasks/create_new_rule';
 import {
   goToExecutionLogTab,
   getExecutionLogTableRow,
   refreshRuleExecutionTable,
   filterByRunType,
 } from '../../../../tasks/rule_details';
-import { getNewRule } from '../../../../objects/rule';
+import { getCustomQueryRuleParams } from '../../../../objects/rule';
 import { EXECUTION_SHOWING } from '../../../../screens/rule_details';
 import { manualRuleRun } from '../../../../tasks/api_calls/backfill';
 
@@ -37,7 +36,9 @@ describe(
       login();
       deleteAlertsAndRules();
       createRule({
-        ...getNewRule(),
+        ...getCustomQueryRuleParams({
+          enabled: true,
+        }),
       }).then((rule) => {
         cy.wrap(rule.body.id).as('ruleId');
       });
@@ -45,11 +46,23 @@ describe(
 
     it('should display the execution log', function () {
       visit(ruleDetailsUrl(this.ruleId));
-      waitForAlertsToPopulate();
       goToExecutionLogTab();
+
+      cy.waitUntil(
+        () => {
+          cy.log('Waiting for execution logs to appear in execution log table');
+          refreshRuleExecutionTable();
+          return getExecutionLogTableRow().then((rows) => {
+            return rows.length > 0;
+          });
+        },
+        { interval: 5000, timeout: 20000 }
+      );
 
       cy.get(EXECUTION_SHOWING).contains('Showing 1 rule execution');
       getExecutionLogTableRow().should('have.length', 1);
+
+      cy.log('Scheduling a manual rule run');
       manualRuleRun({
         ruleId: this.ruleId,
         start: moment().subtract(5, 'm').toISOString(),
@@ -61,14 +74,16 @@ describe(
           cy.log('Waiting for execution logs to appear in execution log table');
           refreshRuleExecutionTable();
           return getExecutionLogTableRow().then((rows) => {
-            return rows.length === 2;
+            return rows.length > 1;
           });
         },
         { interval: 5000, timeout: 20000 }
       );
-      cy.get(EXECUTION_SHOWING).contains('Showing 2 rule executions');
-      filterByRunType('Manual');
 
+      cy.get(EXECUTION_SHOWING).contains('Showing 2 rule executions');
+      getExecutionLogTableRow().should('have.length', 2);
+
+      filterByRunType('Manual');
       getExecutionLogTableRow().should('have.length', 1);
     });
   }

--- a/x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/rule_details/execution_log.cy.ts
+++ b/x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/rule_details/execution_log.cy.ts
@@ -23,8 +23,7 @@ import { getNewRule } from '../../../../objects/rule';
 import { EXECUTION_SHOWING } from '../../../../screens/rule_details';
 import { manualRuleRun } from '../../../../tasks/api_calls/backfill';
 
-// FLAKY: https://github.com/elastic/kibana/issues/184360
-describe.skip(
+describe(
   'Event log',
   {
     tags: ['@ess', '@serverless'],

--- a/x-pack/solutions/security/test/security_solution_cypress/cypress/tasks/api_calls/backfill.ts
+++ b/x-pack/solutions/security/test/security_solution_cypress/cypress/tasks/api_calls/backfill.ts
@@ -27,8 +27,7 @@ export const manualRuleRun = ({
     body: [
       {
         rule_id: ruleId,
-        start,
-        end,
+        ranges: [{ start, end }],
       },
     ],
   });

